src/jmh/java/column/store/query/logs/FilterForLogStringOperations.java

```diff
@@ -0,0 +1,88 @@
+package column.store.query.logs;
+
+import column.store.Utils;
+import column.store.api.column.Column;
+import column.store.api.column.StringColumn;
+import column.store.api.query.Query;
+import column.store.api.query.StringFilter;
+import column.store.api.read.Reader;
+import column.store.api.read.StringColumnReader;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static column.store.api.query.Filter.whereString;
+
+public class FilterForLogStringOperations {
+
+    private static final StringColumn PROCESS_GROUP = Column.forString("dt.entity.process_group_instance-string");
```

Comment:
We will need to replace `.` with `_`, as discussed on slack.

Reply:
oh forgot to replace the identifiers.
yaml file was update